src/core/store/common/tokensByAsset.selectors.ts

```diff
@@ -62,15 +62,16 @@ export const selectDepositTokenOptionsByAsset = createSelector(
             const allowancesMap = userTokensAllowancesMap[address] ?? {};
             return createToken({ tokenData, userTokenData, allowancesMap });
           });
-        let subgraphTokens: TokenView[] = supportedTokens
+        const subgraphTokens: TokenView[] = supportedTokens
           .filter((address) => !!tokensMap[address])
           .map((address) => {
             const tokenData = tokensMap[address];
             const userTokenData = userTokensMap[address];
             const allowancesMap = userTokensAllowancesMap[address] ?? {};
             return createToken({ tokenData, userTokenData, allowancesMap });
           });
-        return unionBy(mainTokens, subgraphTokens, (o) => o.symbol);
+        const sortedSubgraphTokens = sortBy(subgraphTokens, (o) => o.symbol);
```

Comment:
u probs want to do union then sort otherwise u might get A-Z then A-Z again in the same list

Reply:
This was intentional. I want both `mainTokens` (e.g. DAI, ETH, WETH, etc.) to be sorted A-Z and then `subgraphTokens` to separately be sorted A-Z. `mainTokens` should be displayed before `subgraphTokens` in the list.